### PR TITLE
Feature/rabbit hunt

### PR DESCRIPTION
**Pull request by tompsherman**
# Description

Went down the Rabbit Hole marking problems in comments with ERRLOG and noting any actions taken. Fixed errors in the migrations tables. Back and front end connected, but data still not completely firing properly (see line 79 of MatchupContainer component on front end)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [X] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [X] No
- [ ] Not necessary

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] There are no merge conflicts

### CODE REVIEW

File: data/migrations/20201216145909_change_vote_to_integer.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 
 exports.up = function(knex) {
     return knex.schema.table('Votes', table => {
-        table.dropColumn('Vote');
```
Comment (AprilAshby) [inline data/migrations/20201216145909_change_vote_to_integer.js:4]: Nice job sleuthing! This was buried between a million migrations.

File: api/game/gameHelpers/faceoffRetrieval.js [hunk 2]
```diff
@@ -55,6 +55,7 @@ const addSubmissionsToFaceoffs = async (conn, faceoffs) => {
     const s1 = await dbOps.getSubByID(conn, faceoffs[f].SubmissionID1);
     const s2 = await dbOps.getSubByID(conn, faceoffs[f].SubmissionID2);
     // Add formatted submissions to the faceoffs object reference passed into the function
+    //ERRLOG: submission1 instead of submissionID1(??)
```
Comment (AprilAshby) [inline api/game/gameHelpers/faceoffRetrieval.js:58]: Thanks for figuring out where things were breaking. We have a much better idea of where to go from here.

**Review by AprilAshby — APPROVED** [review]
Thanks for tracking down this bug. We'll get voting figured out asap

File: api/game/gameHelpers/faceoffRetrieval.js (hunk 2 above)
Comment (LizDrumm) [reply api/game/gameHelpers/faceoffRetrieval.js:58]: Yes, thank you.

**Review by LizDrumm — APPROVED** [review]
Appreciate your time to find these issues.

**Pull request merged**